### PR TITLE
Vr shop

### PR DESCRIPTION
**Pull request by AlessandroSigna**
This PR brings all scripts for vrShop domain.

To test without issues: https://github.com/highfidelity/hifi/pull/6786
To experience in the current release of interface that PR needs to be merged 
and the scripts need to be updated to the new collisionMask system.

README file available for more info.

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: 41e6dd48e07b4a0cee04d23966be06c3b0fbc11a available at: https://gist.github.com/8ceea6f9abe3f8cb59c2<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/pr-interface-win64-PR-6842.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6842.dmg

File: examples/vrShop/review/shopReviewEntityScript.js [hunk 1]
```diff
@@ -0,0 +1,477 @@
...
+    
+    //This method is the callback called once the recording is loaded into the asset
+    function saveDataIntoDB(url) {
+        // Feed the database
+        var dbObj = getEntityCustomData('infoKey', dataBaseID, null);
+        if(dbObj) {
+            var myName = MyAvatar.displayName ? MyAvatar.displayName : "Anonymous";
+            dbObj.dbKey[dbObj.dbKey.length] = {name: myName, score: scoreAssigned, clip_url: url};
+            setEntityCustomData('infoKey', dataBaseID, dbObj);
+            print("Feeded DB: " + url");
```
Comment (imgntn) [inline examples/vrShop/review/shopReviewEntityScript.js]: i believe there's an extra ``` " ``` here after ``` + url ```  everything after this displays as red on github

**Comment by hifi-gustavo** [comment]
Build console output for commit: 2da5cfd9e2f99c9c549f7bcdfd7415669eb3a882 available at: https://gist.github.com/8ceea6f9abe3f8cb59c2<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/pr-interface-win64-PR-6842.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6842.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 41e6dd48e07b4a0cee04d23966be06c3b0fbc11a available at: https://gist.github.com/8ceea6f9abe3f8cb59c2<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/pr-interface-win64-PR-6842.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6842.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 737a00a2cb2eef2c2e8544f6d2365fc6e596ea4d available at: https://gist.github.com/8ceea6f9abe3f8cb59c2<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/pr-interface-win64-PR-6842.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6842.dmg

**Pull request merged**